pkg/tidb/forwarder.go

```diff
@@ -92,12 +91,7 @@ func (f *Forwarder) pollingForTiDB() {
 			allTiDB, err = topology.FetchTiDBTopology(bo.Context(), f.etcdClient)
 			return err
 		}, bo)
-		if err != nil {
-			if errorx.IsOfType(err, ErrNoAliveTiDB) {
-				f.sqlProxy.updateRemotes(nil)
-				f.statusProxy.updateRemotes(nil)
-			}
-		} else {
+		if err == nil {
```

Comment:
`errorx.IsOfType(err, ErrNoAliveTiDB)` is a legacy code and is always false, so I simply remove entire `err != nil` block